botorch/acquisition/multi_objective/multi_output_risk_measures.py

```diff
@@ -94,6 +94,25 @@ def forward(self, samples: Tensor, X: Optional[Tensor] = None) -> Tensor:
         pass  # pragma: no cover
 
 
+class MultiOutputExpectation(MultiOutputRiskMeasureMCObjective):
+    r"""A multi-output MC expectation risk measure."""
+
+    def forward(self, samples: Tensor, X: Optional[Tensor] = None) -> Tensor:
+        r"""Calculate the expectation of the given samples. Expectation is
+        calculated over each `n_w` samples in the q-batch dimension.
+
+        Args:
+            samples: A `sample_shape x batch_shape x (q * n_w) x m`-dim tensor of
+                posterior samples. The q-batches should be ordered so that each
+                `n_w` block of samples correspond to the same input.
+            X: A `batch_shape x q x d`-dim tensor of inputs. Ignored.
+        Returns:
+            A `sample_shape x batch_shape x q x m`-dim tensor of expectation samples.
+        """
+        prepared_samples = self._prepare_samples(samples)
```

Comment:
since `X` is ignored, should we emit a warning if the passed in `X` is not `None`?

Reply:
`X` is ignored in most MC objectives, including all risk measures. Since we call the objective with `X=X` in the acquisition functions, I feel like this may end up flooding the output with warnings